impl/src/main/java/org/apache/myfaces/flow/cdi/FlowScopeContext.java

```diff
@@ -244,7 +244,7 @@ public <T> T get(Contextual<T> bean, CreationalContext<T> creationalContext)
         List<String> activeFlowMapKeys = getStorageHolder(facesContext).getActiveFlowMapKeys(facesContext);
         for (String flowMapKey : activeFlowMapKeys)
         {
-            ContextualStorage storage = getContextualStorage(facesContext, false, flowMapKey);
+            ContextualStorage storage = getContextualStorage(facesContext, true, flowMapKey);
```

Comment:
Atom decided to fix all the whitespace errors. Here's the NPE fix. true  - createIfNotExist